functests/1_performance/performance.go

```diff
@@ -781,6 +782,19 @@ func validatTunedActiveProfile(nodes []corev1.Node) {
 	var err error
 	var out []byte
 	activeProfileName := components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance)
+
+	// check if some another Tuned profile overwrites PAO profile
+	tunedList := &tunedv1.TunedList{}
+	if err := testclient.Client.List(context.TODO(), tunedList); err != nil {
+		klog.Error(err)
+	}
+	for _, t := range tunedList.Items {
+		if strings.Contains(*t.Spec.Profile[0].Data, fmt.Sprintf("include=%s", activeProfileName)) {
```

Comment:
we should check:
- `len(t.Spec.Profile) > 0`
- `t.Spec.Profile[0].Data != nil`

Reply:
added this checking